controllers/loadtest_controller.go

```diff
@@ -58,6 +58,7 @@ type LoadTestReconciler struct {
 // +kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups="",resources=nodes,verbs=get;list;watch
 // +kubebuilder:rbac:groups="",resources=nodes/status,verbs=get
+// +kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Do we need all of the verbs? `get;list;watch;create;update;patch;delete`

Reply:
Good point, only `create` is observed when controller crashed and restarted. 
I assume we can grant controller all the power to deal with events, no?